charts/sessionspaces/templates/clusterrole.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "sessionspaces.fullname" . }}
```

Comment:
Maybe we should use the `sessionspaces.serviceAccountName` here? Thoughts?

Reply:
Yeah, makes sense!